### ISSUE TITLE
How to calculate average segment duration?

### ISSUE THREAD

**Issue by irajs** [Dash-Industry-Forum/DASH-IF-IOP#250]
"The SegmentTemplate@duration attribute SHALL define the nominal duration of a media segment in timescale units. The average true duration of media segments SHALL match the nominal duration."  in simple addressing section.

We should remove the second SHALL statement, because you can not check the SHALL requirement by the sentence alone. It is checked with %50 deviation requirement.

**Comment by sandersaares** [comment]
I do not understand this comment. Please clarify what you mean.

It seems quite straightforward to check this:

1. Calculate average true duration of media segments.
1. Does the calculated value match the nominal duration?
1. Done

**Comment by irajs** [comment]
how many segments do u take the average over? not defined.

**Comment by sandersaares** [comment]
Yes. It is not defined. The more segments you take, the more accurate your determination will be.

**Comment by irajs** [comment]
again the real shall  is defined by the 50% deviation, correct? if the deviation is satified, then so is this. And IMO having a loose SHALL in terms of computation is not good. we can turn this sentence to a statement of intent.

**Comment by sandersaares** [comment]
Okay, I will reword it to a statement of intent.

**Comment by sandersaares** [comment]
I will just remove this sentence as it seems to already be sufficiently described by other statements.
